src/main/java/codeu/model/data/MentionList.java

```diff
@@ -0,0 +1,99 @@
+package codeu.model.data;
+
+import codeu.model.store.basic.UserStore;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class MentionList {
```

Comment:
In terms of naming, I think "MentionParser" might be more clear, as that's the key thing that this class does -- it goes through a message and identifies all the mentions from it.  A quick javadoc as to your intent would also be helpful in clarifying that you're not just making a fancy List<Mention> (which you're not -- it's much more than that).